samza-sql/src/test/java/org/apache/samza/sql/testutil/SamzaSqlTestConfig.java

```diff
@@ -95,7 +96,7 @@
         ConfigBasedUdfResolver.class.getName());
     staticConfigs.put(configUdfResolverDomain + ConfigBasedUdfResolver.CFG_UDF_CLASSES, Joiner.on(",")
         .join(MyTestUdf.class.getName(), RegexMatchUdf.class.getName(), FlattenUdf.class.getName(),
-            MyTestArrayUdf.class.getName()));
+            MyTestArrayUdf.class.getName(), BuildOutputRecordUdf.class.getName()));
```

Comment:
In Shell, when user performs "show functions", Shell will show all the details of UDFs supported by SamzaSQL. Currently we manually register the UDF info in [listFunctions](https://github.com/apache/samza/blob/master/samza-sql-shell/src/main/java/org/apache/samza/sql/client/impl/SamzaExecutor.java#L323). I added TODO to fix this and make it automatically achieve all UDF details, for now, shall we still manually register the UDF info in  [listFunctions](https://github.com/apache/samza/blob/master/samza-sql-shell/src/main/java/org/apache/samza/sql/client/impl/SamzaExecutor.java#L323)?

Reply:
These are added explicitly for our integration tests. For shell, in future, we will have to add a way to detect all the udfs.